tueplots/axes.py

```diff
@@ -36,9 +37,10 @@ def lines(
         "grid.color": color,
         # Update the linestyle of the grid
         # (it shares a color with the frame, and needs to be distinguishable)
-        "grid.linestyle": "dotted",
+        "grid.linestyle": "solid",
         "grid.alpha": 0.25,
         # Set visibility of spines
+        "axes.axisbelow": axis_below,
```

Comment:
I agree with you that `axis_below` is more visually pleasing than `axisbelow`.
But I think for consistency's sake, it is better to stick to `axisbelow`, because it is what matplotlib prefers.

Reply:
Good point. 

I can do these changes later this week, unless you want to go ahead and do them yourself.

Reply:
> I can do these changes later this week

Thanks :+1: